### PR TITLE
Library upgraded to Android build-tools 23

### PR DESCRIPTION
**Pull request by tsfoer**
As google changed some code with API and build-tools 23, the library has to be updated to reflect the changes

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.2.2'
+        classpath 'com.android.tools.build:gradle:1.3.0'
```
Comment (denisk20) [inline build.gradle:8]: This can be `1.3.1`, right?

Comment (tsfoer) [inline build.gradle:8]: Yes, 1.3.1 is right

**Comment by denisk20** [comment]
@tsfoer I'm sorry but I can't accept current changes: the dialog works incorrectly on Android 6 emulator, the whole color area is white:
image
Without these changes it works fine on the same emulator instance:
image

Can you please take a look why it happens? Otherwise the changes look great.

**Comment by tsfoer** [comment]
Strange, it was working on my test device. I'll have a look as soon as possible on it.

**Comment by tsfoer** [comment]
So i didn't manage to get it working on Android 6. My previous success seemed to be some gradle compile cache mix construct. After doing a full project clean it keeps showing the white area.
Seems that ComposeShader() either has some serious problems or something else has changed in Android 6 that I'm not aware of.

**Comment by denisk20** [comment]
I'll need to investigate it. Thank you for your research!

**Comment by stefan2606** [comment]
buildToolsVersion "23.0.1"

**Comment by denisk20** [comment]
@stefan2606 it works! Thanks so much for pointing that out!
@tsfoer I'm merging your pull request and will update tools to 23.0.1 in a separate commit.

**Pull request merged**